<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePasswordWindow</class>
 <widget class="QDialog" name="changePasswordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>change password</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sources.qrc">
    <normaloff>:/Images/user (1).png</normaloff>:/Images/user (1).png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>750</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="sources.qrc">:/Images/template_change password.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>606</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background : transparent;
border-radius:10px;
}

QPushButton::hover{
background:rgba(255, 255, 255, 50);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDarea">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>431</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background:transparent;
border-radius:45px;
padding:15px;
	font: 700 10pt &quot;Segoe UI&quot;;
color:rgb(143, 142, 138);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter you ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldPass_area">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>277</y>
     <width>391</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background:transparent;
border-radius:45px;
padding:15px;
	font: 700 10pt &quot;Segoe UI&quot;;
color:rgb(143, 142, 138);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter you old password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPass_area">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>391</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background:transparent;
border-radius:45px;
padding:15px;
	font: 700 10pt &quot;Segoe UI&quot;;
color:rgb(143, 142, 138);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter you new password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="againNewPass_area">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>508</y>
     <width>391</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background:transparent;
border-radius:45px;
padding:15px;
	font: 700 10pt &quot;Segoe UI&quot;;
color:rgb(143, 142, 138);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter you new password again</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showOldPass">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background : transparent;
border-radius:10px;
}

QPushButton::hover{
background:rgba(255, 255, 255, 50);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="showNewPass">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background : transparent;
border-radius:10px;
}

QPushButton::hover{
background:rgba(255, 255, 255, 50);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="showAgainNewPass">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>530</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background : transparent;
border-radius:10px;
}

QPushButton::hover{
background:rgba(255, 255, 255, 50);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
